{{ cookiecutter.project_slug }}/api/pyproject.toml

```diff
@@ -1,12 +1,5 @@
-[build-system]
-requires = ["poetry-core", "setuptools", "pip"]
```

Comment:
It seems like we can drop this for standalone application  (`package-mode = false`), but let me know if you disagree.